src/main/scala/com/raphtory/core/config/telemetry/QueryTelemetry.scala

```diff
@@ -0,0 +1,114 @@
+  *
+  *
+  */
+object QueryTelemetry {
+
+  val conf = new ConfigHandler().getConfig
+
+  val readyCount: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_ready_count")
+      .help("Ready count for query handler ")
+      .register
+
+  val vertexCount: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_vertex_count")
+      .help("Total vertex count for query handler")
+      .register
+
+  val receivedMessageCount: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_msgs_received")
+      .help("Total messages received by Query Handler")
+      .register
+
+  val sentMessageCount: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_msgs_sent")
+      .help("Total messages sent by Query Handler")
+      .register
+
+  val totalTableOperations: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_table_operations")
+      .help("Total table operations by Query Handler")
+      .register
+
+  val totalGraphOperations: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_graph_operations")
+      .help("Total graph operations by Query Handler")
+      .register
+
+  val totalPerspectivesProcessed: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_handler_perspective_processed")
+      .help("Total perspectives processed by Query Handler")
+      .register
+
+  val totalPointQueriesSpawned: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_manager_point_query_count")
+      .help("Total point queries by query manager")
+      .register
+
+  val totalRangeQueriesSpawned: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_manager_range_query_count")
+      .help("Total range queries by query manager")
+      .register
+
+  val totalLiveQueriesSpawned: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_manager_live_query_count")
+      .help("Total live queries by query manager")
+      .register
+
+  val totalQueryManagerWatermarks: Counter =
+    Counter
+      .build
+      .namespace(conf.getString("raphtory.prometheus.query_namespace"))
+      .name("query_manager_watermarks_count")
+      .help("Total watermarks created by query manager")
+      .register
+
+  val totalQueriesSpawned: Counter =
```

Comment:
Can probably just have this one instead of the total of each type above. Possibly have a total running queries could also be a bit more useful